### PR TITLE
Remove hardcoded regions

### PR DESCRIPTION
**Pull request by bonclay7**
### What does this PR do?

AWS regions settings were hardcoded in some samples, confusing some users

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [x] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/observability/adot-amp-grafana-for-haproxy/variables.tf [hunk 1]
```diff
@@ -10,3 +10,8 @@ variable "grafana_api_key" {
   default     = ""
   sensitive   = true
 }
+
+variable "aws_region" {
+  description = "AWS Region to deploy resources"
+  type        = string
+}
```
Comment (askulkarni2) [inline examples/observability/adot-amp-grafana-for-haproxy/variables.tf:17]: Provide `default = us-west-2` so our CI doesn't wait for input.

**Review by askulkarni2 — CHANGES_REQUESTED** [review]
Need `default = us-west-2` for our CI checks.

File: examples/observability/adot-amp-grafana-for-haproxy/variables.tf (hunk 1 above)
Comment (bonclay7) [reply examples/observability/adot-amp-grafana-for-haproxy/variables.tf:18]: Thanks for the review! Can you please merge if that's good for you?

**Review by kcoleman731 — APPROVED** [review]

**Review by kcoleman731 — APPROVED** [review]

**Review by askulkarni2 — APPROVED** [review]
LGTM!

**Pull request merged**